main-settings/src/main/scala/sbt/dsl/LinterLevel.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.dsl
+
+sealed trait LinterLevel
```

Comment:
I considered contraband for this, but wasn't sure if there is a way to put the (prioritized) implicit instances in the companion object.

Reply:
You can use https://www.scala-sbt.org/contraband/schema.html#Extra+code

```
  #xcompanioninterface CompanionInterface1
```